theme/ckeditor5-media-embed/mediaform.css

```diff
@@ -38,7 +38,7 @@
 			}
 
 			/* Let the long error messages wrap in the narrow form. */
-			& .ck-labeled-input__error {
+			& .ck-labeled-field-view__error {
```

Comment:
Shouldn't it also work for non-error statuses?

![image](https://user-images.githubusercontent.com/1099479/78667142-b6997200-78d8-11ea-8dec-938c9b649edd.png)